exec/java-exec/src/main/java/org/apache/drill/exec/work/WorkManager.java

```diff
@@ -442,6 +467,14 @@ public void run() {
       // StatusThread is started
       final Controller controller = dContext.getController();
       final DrillbitEndpoint localBitEndPoint = dContext.getEndpoint();
+      try {
+        pollShutdown(drillbit);
+      } catch (IOException e) {
+        e.printStackTrace();
```

Comment:
I think we should not use `e.printStackTrace();` but rather do proper error handling or logging.

Reply:
Changed it.